src/Shared/Components/Button/Button.component.tsx

```diff
@@ -154,10 +154,14 @@ const Button = ({
             }
         }
 
-        return {
-            alwaysShowTippyOnHover: showTooltip && !!tooltipProps?.content,
-            ...tooltipProps,
+        if (!tooltipProps.shortcutKeyCombo) {
+            return {
+                alwaysShowTippyOnHover: showTooltip && !!tooltipProps?.content,
+                ...(tooltipProps as Omit<TooltipProps, 'shortcutKeyCombo' | 'showOnTruncate'>),
+            }
         }
+
+        return tooltipProps
```

Comment:
Please verify if alwaysShowTippyOnHover is intended to be false here

Reply:
No need for this anymore